core/test/test-helpers.js

```diff
@@ -14,62 +14,81 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { fixture, fixtureCleanup, html } from '@open-wc/testing-helpers';
+import * as utils from '@polymer/polymer/lib/utils/flush.js';
 
 /* eslint-disable no-unused-vars,no-param-reassign */
 
 window.fetch = null;
 
-function timePasses(ms) {
+function timePasses(ms = 1) {
   return new Promise((resolve) => {
     window.setTimeout(() => {
       resolve();
-    }, ms || 1);
+    }, ms);
   });
 }
 
-const flushCb = window.flush;
-window.flush = () =>
-  new Promise((resolve) => {
-    flushCb(() => {
-      if (window.requestIdleCallback) {
-        window.requestIdleCallback(() => resolve());
-      } else {
-        window.setTimeout(() => resolve(), 16);
-      }
-    });
-  });
+const flush = async () => {
+  utils.flush();
+  await timePasses(0);
+};
 
-function waitForEvent(el, event, times) {
-  times = times || 1;
+function waitForEvent(el, event, times = 1) {
   return new Promise((resolve) => {
     const listener = (e) => {
       if (--times === 0) {
-        el.removeEventListener(e, listener);
+        el.removeEventListener(event, listener);
         resolve(e);
       }
     };
     el.addEventListener(event, listener);
   });
 }
 
-function waitChanged(el, prop, times) {
+function waitChanged(el, prop, times = 1) {
   return waitForEvent(el, `${prop}-changed`, times).then((e) => e.detail);
 }
 
-function login(server, loginFixture) {
-  server.respondWith('GET', '/dummy/json/cmis', [200, { 'Content-Type': 'application/json' }, '{}']);
+let server;
+
+async function login() {
+  server = sinon.fakeServer.create();
+  server.autoRespond = true;
 
-  server.respondWith('POST', '/dummy/api/v1/automation/login', [
+  server.respondWith('GET', '/json/cmis', [200, { 'Content-Type': 'application/json' }, '{}']);
```

Comment:
assuming we were exporting the responseHeaders:
```suggestion
  server.respondWith('GET', '/json/cmis', [200, responseHeaders.json, '{}']);
```